rust/main/agents/relayer/src/msg/metadata/base.rs

```diff
@@ -44,17 +47,27 @@ pub enum MetadataBuilderError {
     MaxDepthExceeded(u32),
 }
 
+#[derive(Clone, Debug)]
+pub enum Metadata {
+    /// Able to fetch metadata
+    Ok(Vec<u8>),
```

Comment:
open to `Some` or another name

Reply:
`Success`

Reply:
went for `Found` since the existence of metadata doesn't guarantee it's successfull (e.g. non-malicious)